src/wp-includes/class-wp-theme-json.php

```diff
@@ -2283,11 +2310,11 @@ public function merge( $incoming ) {
 		$nodes        = static::get_setting_nodes( $incoming_data );
 		$slugs_global = static::get_default_slugs( $this->theme_json, array( 'settings' ) );
 		foreach ( $nodes as $node ) {
-			$slugs_node = static::get_default_slugs( $this->theme_json, $node['path'] );
-			$slugs      = array_merge_recursive( $slugs_global, $slugs_node );
-
 			// Replace the spacing.units.
-			$path    = array_merge( $node['path'], array( 'spacing', 'units' ) );
+			$path   = $node['path'];
```

Comment:
Why not just use an array declaration here instead of three individual assignments?

```
$path = array(
	$node['path'],
	'spacing',
	'units'
);
```

Reply:
Your suggested code would not be the same result. `$node['path']` is an array, so in your code the first element would be an array and the other two would be strings, so that would not be correct I think.

Reply:
Ah, yes. You're right on this one. I think a few others it may result in the same. But not this one.